petstore/clients/dotnet/tests/PetsClientTests.cs

```diff
@@ -1,12 +1,22 @@
+using System.ClientModel;
+using System.Text.Json;
+
 namespace PetStore.Tests
 {
     public class PetsClientTests
     {
+        private Pets _petsClient;
+
+        [OneTimeSetUp]
+        public void Setup()
+        {
+            _petsClient = new PetStoreClient(new Uri("http://localhost:5118")).GetPetsClient();
```

Comment:
can we put this Uri value into a config file? This is likely to be shared by server, client and tests.

Reply:
this is only temporary.
I think it is no point that we put it into a config or something.
In the future, there could be some script that assigns this value.